code/modules/admin/topic.dm

```diff
@@ -2633,6 +2633,57 @@
 					newname = "Admin"
 				D.name = newname
 				D.real_name = newname
+			//False flags and bait below. May cause mild hilarity or extreme pain
+			if("fakebiohazard") //GUISE WE HAVE A BLOB
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEB")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Biohzard Alert.")
+				var/choice = input("Set the level of the biohazard alert, or leave at 0 to have a random level (1 to 7 supported only)", "Space FEMA Readiness Program", 0) as num
+				if(!choice || choice > 7 || choice < 0)
+					return
+				biohazard_alert(level = choice)
+			if("fakelifesigns") //MUH ALIUMS
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEL")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Lifesign Alert.")
+				command_alert("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert")
+				world << sound('sound/AI/aliens.ogg')
+			if("fakedelta") //BLOW EVERYTHING
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKET")
+				var/choice = input("Do you wish to include the Hostile Runtimes warning to have an authentic malfunction takeover alert ?") in list("Yes", "No")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Malfunction Takeover Alert (Hostile Runtimes alert [choice == "Yes" ? "included":"excluded"])")
+				if(choice == "Yes")
+					command_alert("Hostile runtimes detected in all station systems, please deactivate your AI to prevent possible damage to its morality core.", "Anomaly Alert")
+				world << "<font size=4 color='red'>Attention! Delta security level reached!</font>" //Don't ACTUALLY set station alert to Delta to avoid fucking shit up for real
+				world << "<font color='red'>[config.alert_desc_delta]</font>"
+				world << sound('sound/AI/aimalf.ogg') //AI got valid
+			if("fakebooms") //Micheal Bay is in the house !
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEE")
+				var/choice = input("How much high-budget explosions do you want ?", "Micheal Bay SFX Systems", 1) as num
+				if(choice > 0) //No negative or null explosion amounts here math genius

```

Comment:
Always try to give feedback to the user when invalidating their input, just print them something like that comment